app/controllers/sql-lite-service.js

```diff
@@ -135,24 +136,24 @@ module.exports = function (app) {
     /**
      * public methods
      */
-    controller.prototype.init = () => {
-        let promises = [];
-        promises.push(Country.init());
-        promises.push(Document.init());
-        promises.push(IdAttributeType.init());
-        promises.push(Token.init());
-        promises.push(Wallet.init());
-        promises.push(AppSetting.init());
-        promises.push(GuideSetting.init());
-        promises.push(IdAttribute.init());
-        promises.push(TokenPrice.init());
-        promises.push(createWalletTokens());
-        promises.push(TransactionHistory.init());
-        promises.push(ActionLog.init());
-        promises.push(WalletSetting.init());
-        promises.push(ExchangeDataHandler.init());
-        return Promise.all(promises)
-    }
+    // controller.prototype.init = () => {
```

Comment:
This can be deleted.

Reply:
thanks, will delete all commented code next commit